src/services/chain-checker.ts

```diff
@@ -201,24 +209,24 @@ export class ChainChecker {
     ] = await Promise.all(promiseStack)
 
     for (const rawNodeChainLog of rawNodeChainLogs) {
-      if (typeof rawNodeChainLog === 'object' && rawNodeChainLog.chainID !== '') {
+      if (typeof rawNodeChainLog === 'object' && ((rawNodeChainLog.chainID as unknown) as string) !== '') {
```

Comment:
Nice

Reply:
so, change it or not? 😂😂

Reply:
Yes it's fine, just saying I learned something